pagopa/pn-frontend
# PN-9455 - Source and first time opening on SEND_NOTIFICATION_DETAIL

## Short description
Added source and first time opening on mixpanel event SEND_NOTIFICATION_DETAIL

## List of changes proposed in this pull request
- Added source (QRcode or LISTA_NOTIFICHE)
- Added first_time_opening (true if is the first time that we're opening a notification)

## How to test
Try to open a notification and on console you should see a log with SEND_NOTIFICATION_DETAIL. Inside this object you should find source and first_time_opening
